.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024, Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Lint
```

Comment:
As for me, the name is too generic. I would be more specific and name it something like "Check Docker Hashes Consistency". So when it's red, we immediately see what exactly failed.

Reply:
I thought we might add other linters in the future. Or do you want to introduce a new GH action for every new linter?

Reply:
I find it cleaner to have a named action per linter/check so the failed action is easier to understand.

Reply:
Renamed it.